dcpy/utils.py

```diff
@@ -0,0 +1,20 @@
+import subprocess
```

Comment:
should this be a folder maybe? Or either way is `utils` too generic? Could have this in a file/folder called `git` as well. 

About the actual contents - git username requires using git command line as far as I know. Not used in this codebase at the moment but is in data library. But for git_branch, should we maybe go with a different approach? Maybe in CI, use available environment variables rather than subprocess? Or either way, we could parse from `.git` folder. Curious on y'all's thoughts

Reply:
I have a weak preference would be for a git.py file. Or just using a [git python package](https://gitpython.readthedocs.io/en/stable/intro.html)? But this seems fine. 

> CI, use available environment variables rather than subprocess

I like this idea